app/views/content_items/coming_soon.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= content_for :page_class, "coming-soon" %>
+<%= content_for :title, "Coming soon - GOV.UK" %>
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <%= render 'govuk_component/title', title: 'Coming soon' %>
+
+    <p class="summary">
+      This document will be published on<br />
+      <%= content_tag(:abbr, class: 'publish_time', title: @content_item.iso8601_publish_time) do %>
```

Comment:
Whitehall was using an older (pre HTML5) way to markup dates and times. You should be able to use the [time tag](http://www.w3.org/TR/html-markup/time.html) now. Which would make this line:

```erb
<%= content_tag(:time, class: 'publish_time', datetime: @content_item.iso8601_publish_time) do %>
```